trackpy/preprocessing.py

```diff
@@ -37,7 +37,7 @@ def ifftn(a):
         return pyfftw.interfaces.numpy_fft.ifftn(a)
 
 
-def bandpass(image, lshort, llong, threshold=None):
+def bandpass(image, lshort, llong, threshold=None, analyze_background=False):
```

Comment:
I would split this up into three functions, on `_bandpass_raw` which returns `signal` and `result` and then two public functions `bandpass` which just calls the `where` and `bandpass_bkgnd_stats` which also does the background computations.